NetCord/MessagePollResults.cs

```diff
@@ -0,0 +1,11 @@
+﻿using NetCord.JsonModels;
+
+namespace NetCord;
+
+public class MessagePollResults(JsonMessagePollResults jsonModel) : IJsonModel<JsonMessagePollResults>
+{
+    JsonMessagePollResults IJsonModel<JsonMessagePollResults>.JsonModel => jsonModel;
+    public bool IsFinalized => jsonModel.IsFinalized;
+    public IReadOnlyList<MessagePollAnswerCount>? Answers = jsonModel.Answers?.Select(x => new MessagePollAnswerCount(x)).ToArray();
```

Comment:
I think a normal constructor with an `if` statement should be used instead of the `?`, that is what NetCord does to prevent setting null to null unnecessarily.